core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -87,7 +87,7 @@ class DynamicBrokerReconfigurationTest extends QuorumTestHarness with SaslSetup
 
   private val servers = new ArrayBuffer[KafkaBroker]
   private val numServers = 3
-  private val numPartitions = 10
+  private val numPartitions = 25
```

Comment:
Is there any satisfactory explanation for the flaky and solution?

Reply:
@dengziming I’ve outlined the possible root cause in the Jira ticket. This change aims to ensure that all fetch threads have tasks as much as possible. However, it’s not an ideal solution since the test remains non-deterministic. we need to dig in it :(